src/Microsoft.Data/BaseColumn.Computations.cs

```diff
@@ -0,0 +1,78 @@
+﻿
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Generated from BaseColumn.Computations.tt. Do not modify directly
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Data
+{
+    public abstract partial class BaseColumn
+    {
+        #region Computations
+        public virtual void Abs()
+        {
+            throw new NotImplementedException();
+        }
+
+        public virtual bool All()
+        {
+            throw new NotImplementedException();
+        }
+
+        public virtual bool Any()
+        {
+            throw new NotImplementedException();
+        }
+
+        public virtual void CumulativeMax()
```

Comment:
Do these Cumulative functions not need to return results?

Reply:
Pandas does not, not that we need to follow Pandas. Arguably this is inconsistent with Max(), which returns the Max value in the column. I would rather consider this [here](https://github.com/dotnet/corefxlab/issues/2680)